nin/backend/compile.js

```diff
@@ -63,9 +64,21 @@ var compile = function(projectPath, options) {
         fs.writeFileSync(projectPath + '/bin/' + filename, data);
       });
     }
+
+    var settings = projectSettings.load(projectPath);
+    var metadata = [
```

Comment:
It's mainly in order for the metadata tEXt to look nicer with the title first. When generating the OG tags I think I might switch to a dictionary, as i need to reference the authors field directly then.